dashboard/app/controllers/pd/workshops_controller.rb

```diff
@@ -0,0 +1,14 @@
+module Pd
```

Comment:
This file was unchanged other than the rename and adding the comment, but I think because enough lines were different it thought I deleted the previous file and added in this new one.